main.go

```diff
@@ -91,10 +93,45 @@ func (o *Options) Complete() error {
 	if o.endDay != 0 {
 		o.NumDays = o.endDay - o.StartDay
 	}
+	for _, openshiftRelease := range o.OpenshiftReleases {
+		o.Dashboards = append(o.Dashboards, dashboardArgFromOpenshiftRelease(openshiftRelease))
+	}
 
 	return nil
 }
 
+func (o *Options) ToTestGridDashboardCoordinates() []sippyserver.TestGridDashboardCoordinates {
+	dashboards := []sippyserver.TestGridDashboardCoordinates{}
+	for _, dashboard := range o.Dashboards {
+		tokens := strings.Split(dashboard, "=")
+		if len(tokens) != 3 {
+			// launch error
+			panic(fmt.Sprintf("must have three tokens: %q", dashboard))
+		}
```

Comment:
could we move this to validate function? making sure the Dashboard is always in the format we need?

Reply:
> could we move this to validate function? making sure the Dashboard is always in the format we need?

I would still keep this panic, but sure, adding it to validate is reasonable.